addon/HomepageStudy.jsm

```diff
@@ -13,11 +13,11 @@ const EXPORTED_SYMBOLS = ["HomepageStudy"];
 Cu.import("resource://gre/modules/Services.jsm");
 Cu.import("resource://gre/modules/Log.jsm");
 
+const PREF_HOMEPAGE = "browser.startup.homepage";
```

Comment:
Let's not move things around unless needed, otherwise it just makes it hard when investigating the history of the code. Move this back to where it was.

Reply:
Done

Reply:
Done